build-apps.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Why do we need this? We have cmake as build system already.

Reply:
You need this if you want to build the docker image locally, for testing purposes or because your user/group id is not 1000 so you cannot open the output files without changing the permissions after each run.

Reply:
But you can add this command to the cmake files.

Reply:
No, this is an optional more high level command (docker...) to actually run a build _using_ the cmake files _inside_ without the need of configuring your system.